llvm/unittests/Analysis/DXILResourceTest.cpp

```diff
@@ -64,7 +64,11 @@ testing::AssertionResult MDTupleEq(const char *LHSExpr, const char *RHSExpr,
                                      << "Which is:\n"
                                      << "  " << RHSRepr;
 }
+
 #define EXPECT_MDEQ(X, Y) EXPECT_PRED_FORMAT2(MDTupleEq, X, Y)
+
+#define EXPECT_PREQ(X, Y, Z) EXPECT_EQ(X, (std::pair<uint32_t, uint32_t>{Y, Z}))
```

Comment:
It took me a second to map "PR" to "pair", maybe the slightly more explicit "EXPECT_PAIR_EQ" would be a better name?

Reply:
Oh, PR might be "Props", if that's the case I guess "EXPECT_PROPS_EQ". In any case, PR isn't obvious to me.